spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextResourceLoaderAutoConfiguration.java

```diff
@@ -29,6 +30,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
This one definitely stops me from starting my app in my local, non-AWS environment, ultimately because region auto-detection is being used (by default). I'm around today on gitter if you want to discuss.